### ISSUE TITLE
Batch/Script Mode for git gc

### ISSUE THREAD

**Issue by nathanila** [git/git-scm.com#1360]
This is an issue about git itself.  I went to https://git-scm.com/community and the link on the page directed me here.  If this is the wrong place, please point me in the right direction.

Please create a batch/script mode for `git gc`.  I setup my laptop to automatically run every morning `git gc --aggressive` on a repository.  `git gc` prompts me everyday with the following prompt.

> Unlink of file '.git/objects/pack/pack-b80fde308e595564a2196cc2d2d1264e74a04330.pack' failed. Should I try again? (y/n)

I would like `git gc` to output that it was unable to unlink the file and continue.  I would then setup my laptop to automatically run `git gc` whenever my laptop goes idle.  Thus, I get the benefits of `git gc` throughout the day.

**Comment by peff** [comment]
Questions about Git itself should generally be directed to the mailing list described on the community page. However, the prompt in question is actually a Windows-specific thing; you might have better luck asking in the [git-for-windows issue tracker](https://github.com/git-for-windows/git/issues).

**Comment by jnavila** [comment]
I'm curious why you would want to run `git gc --aggressive` so often without letting git manage the db by itself.

**Comment by nathanila** [comment]
In the past, I never ran `git gc`.  I was having some git performance problems and read about `git gc`.  `git gc` helped the performance.  So, I scheduled my machine to run `git gc --aggressive` each morning before I wake up.  

If `git gc` could run without human input, I figured why not run `git gc` when my machine is idle.  I am clearly not using the machine and I will get the best performance possible out of git... maybe I am being obsessive about performance.
